File: Practice/Financial Calculator.py
#MY, Financial Calculator
print("mortal, who seekith financial prosparity, answer the fallowing qustions with numbers or thy seed shall be cursed with redundancy and the eternal status of 'jobless'")

rent = float(input("what is thy rent:USD "))

utu = float(input("what is thy utilities bill:USD "))

grow = float(input("what is thy groceries bill:USD "))

tra = float(input("what is you transport expence amount:USD "))

total_income = float(input("what is thy income:USD "))

total_loss = grow + utu + rent + tra



rent_p = int((rent/total_income) * 100)

utu_p = int((utu/total_income)*100)
# half way through I found a more varible less way to do this but didn't want to update what I made to it

print("thy rent is $", rent, "which is", rent_p, "precent of thy income.")

print("thy utilities bill is $", utu, "which is", utu_p, "precent of thy income.")

print("thy groceries bill is $", grow, "which is", int((grow/total_income) * 100), "precent of thy income.")

print("thy transport expence is $", tra, "which is", int((tra/total_income) * 100), "precent of thy income.")


sav = round((total_income * 0.1), 2)

spend = round((total_income - (sav + total_loss)), 2)

print("Mortal, after paying these expences, you have $", total_income-total_loss, "left, which is", round(((total_loss/total_income)*100),2), "precent of you income. It is advised you add at lest $", sav, "to your savings and set $", spend, "as your spending amount, heed these words or face a curse. NOW BEGONE! ")

File: C/notes/practice/Financial_calculator.c
//MY P6 Financial_Calculator

#include <stdio.h>

#include <math.h>

int main(){

    float inco;
    float rent;
    float util;
    float groc;
    float tran;
    
    
    printf("What is thy income?:$ ");
    scanf("%f", &inco);

    printf("\nWhat is thy rent?: ");
    scanf("%f", &rent);

    printf("\nWhat is thy utilities?:$ ");
    scanf("%f", &util);

    printf("\nWhat is thy groceries:$ ");
    scanf("%f", &groc);

    printf("\nWhat is thy transportation expence?:$ ");
    scanf("%f", &tran);

    printf("\nYour rent is $%.2f and that is %.2f precent of your income", rent, (rent/inco)*100);

    printf("\nYour utilities is $%.2f and that is %.2f precent of your income", util, (util/inco)*100);

    printf("\nYour groceries is $%.2f and that is %.2f precent of your income", groc, (groc/inco)*100);

    printf("\nYour transportation expences sum to $%.2f, which is %.2f precent of your income", groc, (groc/inco)*100);

    printf("It is advised you add $%.2f to your savings", inco * 0.1);

    prinft("This leaves you with $%.2f to spend", inco-((groc+util+rent+tran)+ inco * 0.1 ));
    //PLEASE TEACH US HOW TO WRITE FUNCTIONS ALREADY
    return 0;

}